webapp/src/main/webapp/vue-apps/Admin/components/AdminApp.vue

```diff
@@ -102,27 +102,35 @@ export default {
     return {
       providers: [],
       switcher: false,
-      error: null
+      error: null,
+      log: null
     };
   },
   created() {
+    this.log = webConferencing.getLog().prefix("webconferencing.admin");
     this.getProviders();
   },
   methods: {
     async getProviders() {
       // services object contains urls for requests
       try {
-        // const data = await getData(this.services.providers);
         const response = await webConferencing.getProvidersConfig();
         const data = webConferencing.getProvidersConfig().done(response);
         this.error = null;
         this.providers = await response;
```

Comment:
Request in single await line to get rid of not required variable `response`

Reply:
We have changed the logic and don't use 'await'